packages/vscode-extension/package.json

```diff
@@ -232,11 +232,29 @@
         },
         {
           "command": "fx-extension.viewEnvironment",
-          "when": "view == teamsfx-environment && viewItem =~ /environment/"
+          "when": "view == teamsfx-environment && viewItem =~ /environment.*/"
         },
         {
           "command": "fx-extension.viewEnvironmentWithIcon",
-          "when": "view == teamsfx-environment && viewItem =~ /environment/",
+          "when": "view == teamsfx-environment && viewItem =~ /environment.*/",
+          "group": "inline@3"
+        },
+        {
+          "command": "fx-extension.preview",
+          "when": "view == teamsfx-environment && viewItem =~ /environment-provisioned/"
```

Comment:
Will tree view automatically refresh after provisioned?

Reply:
after deployment? otherwise user will got 404 when preview

Reply:
> Will tree view automatically refresh after provisioned?

Yes, the tree view will auto refresh

Reply:
> after deployment? otherwise user will got 404 when preview. Not sure if is there any way we can detect whether an app is deployed before?

![image](https://user-images.githubusercontent.com/49138419/142344488-eec286c1-66bc-4a33-b397-37d040bd66ee.png)
![image](https://user-images.githubusercontent.com/49138419/142344567-0cb11ca7-ebfe-4202-9d80-f06c1814fac6.png)

Reply:
I have an idea but may change frontend provision behavior - create an default page at first provision.
(Similar to create new app service, after creating new app service without any code deployed, the homepage shows a welcome page instead of 404)